internal/crossdock/client/echo/protobuf.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package echo
+
+import (
+	"context"
+	"time"
+
+	disp "go.uber.org/yarpc/internal/crossdock/client/dispatcher"
+	"go.uber.org/yarpc/internal/crossdock/client/random"
+	"go.uber.org/yarpc/internal/crossdock/crossdockpb"
+
+	"github.com/crossdock/crossdock-go"
+)
+
+// Protobuf implements the 'protobuf' behavior.
+func Protobuf(t crossdock.T) {
+	ProtobufTransport(t, "")
+}
+
+// ProtobufTransport implements the 'protobuf' behavior for the given transport or behavior transport.
+func ProtobufTransport(t crossdock.T, transport string) {
```

Comment:
Not wild about the naming of this func... think you can introduce a verb in here?